themes/vpge/lib/scss/components/grad-grow-filter/_grad-grow-filter.scss

```diff
@@ -192,6 +193,37 @@
                 padding: 0.2rem 1.7rem;
                 font-weight: 700;
                 width: fit-content;
+            }
+
+              // Career Development
+              [data-id="1196"] {
+                color: $vpge-compentency-color-purple;
+                border: .2rem solid $vpge-compentency-color-purple;
+              }
+              // Communication
```

Comment:
Need a spacer line between each of these new blocks of code.

Reply:
Done